cmd/hz/internal/thrift/tags.go

```diff
@@ -93,7 +93,7 @@ var (
 )
 
 func CheckTagOption(args *config.Argument) []generator.Option {
-	ret := []generator.Option{}
+	var ret []generator.Option
```

Comment:
May you give some more info or context about this change?

Reply:
<img width="860" alt="image" src="https://user-images.githubusercontent.com/27986239/199926184-dc473b0f-dec0-4c87-863e-d8fa2186830f.png">
It's only used in one place and it doesn't affect other places

Reply:
see: https://github.com/cloudwego/hertz/blob/b0bcce3a032cf0e450677ab8d93138f273d46492/cmd/hz/internal/generator/model.go#L89

Reply:
I think [welkeyever](https://github.com/welkeyever) wants to know what problem this PR is trying to solve, why the codes are formatted and the reason and context ?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices